docs/architecture/modern-web-apps-azure/common-client-side-web-technologies.md

```diff
@@ -3,7 +3,7 @@ title: Common client side web technologies
 description: Architect Modern Web Applications with ASP.NET Core and Azure | Common client side web technologies
 author: ardalis
 ms.author: wiwagn
-ms.date: 01/30/2019
+ms.date: 12/4/2019
 ---
 # Common client side web technologies
```

Comment:
Some Acrolinx errors found. I am unable to get to the exact statement because those existed prior to your change. Can you fix these:
- replace all occurrences of `client side` with `client-side`
- `generally recommended` to just `recommended`
-  `a lot of` is used multiple times. Probably, replace with `numerous`, `many` or `much`
- `Are new projects building built with it?` remove `building`
- replace `well-documented` with `well documented`